.eslintrc.yaml

```diff
@@ -110,6 +110,9 @@ rules:
   }, {
     selector: "ThrowStatement > CallExpression[callee.name=/Error$/]",
     message: "Use new keyword when throwing an Error."
+    }, {
+    selector: "CallExpression[callee.object.name='assert'][callee.property.name='fail'][arguments.length=1]",
+    message: "assert.fail() message should be third argument"
```

Comment:
This rule came out of fixing a bunch of actual misuse in the code base. In every single case, it was `assert.fail(string)` so I'm pretty OK with it the way it is. I can't even fathom what someone would mean by `assert.fail(false)` to be honest.